### ISSUE TITLE
setup.cfg - Don't build universal wheels

### ISSUE THREAD

**Issue by darkrain42** [pygments/pygments#1410]
Since 2.6.0 drops support for Python 2, please don't build universal wheels

https://github.com/pygments/pygments/blob/7ead1f6ce87dfc792bd3ae929f51b88973cc6a79/setup.cfg#L9-L10

Trying to install Pygments from a local offline cache that includes both Python 2 and Python3 packages fails with "ERROR: Package 'pygments' requires a different Python: 2.7.17 not in '>=3.5'"

```bash
paul@host tmp.ua1Eckz4Id % python2 -m pip wheel pygments && python3 -m pip wheel pygments
Collecting pygments
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/be/39/32da3184734730c0e4d3fa3b2b5872104668ad6dc1b5a73d8e477e5fe967/Pygments-2.5.2-py2.py3-none-any.whl
  Saved ./Pygments-2.5.2-py2.py3-none-any.whl
Skipping pygments, due to already being wheel.
Collecting pygments
  Downloading https://files.pythonhosted.org/packages/56/7c/3551e7e8cd399d9dc1a3e4f44affc2c7c8680c5ad60436ede3856f2c2020/Pygments-2.6.1-py2.py3-none-any.whl (914kB)
    100% |████████████████████████████████| 921kB 2.1MB/s
  Saved ./Pygments-2.6.1-py2.py3-none-any.whl
Skipping pygments, due to already being wheel.

paul@host tmp.ua1Eckz4Id % virtualenv -p python2 . && . ./bin/activate && pip install --upgrade pip setuptools wheel
...
(tmp.ua1Eckz4Id) paul@host tmp.ua1Eckz4Id % pip install --no-index --find-links ./ pygments
Looking in links: ./
Processing ./Pygments-2.6.1-py2.py3-none-any.whl
ERROR: Package 'pygments' requires a different Python: 2.7.17 not in '>=3.5'
(tmp.ua1Eckz4Id) paul@host tmp.ua1Eckz4Id % pip install --no-index --find-links ./ -vvv pygments
...
Given no hashes to check 2 links for project 'pygments': discarding no candidates
Using version 2.6.1 (newest of versions: 2.5.2, 2.6.1)
Processing ./Pygments-2.6.1-py2.py3-none-any.whl
  Added pygments from file:///tmp/tmp.ua1Eckz4Id/Pygments-2.6.1-py2.py3-none-any.whl to build tracker '/tmp/pip-req-tracker-u4Z3dw'
  Removed pygments from file:///tmp/tmp.ua1Eckz4Id/Pygments-2.6.1-py2.py3-none-any.whl from build tracker '/tmp/pip-req-tracker-u4Z3dw'
Cleaning up...
Removed build tracker: '/tmp/pip-req-tracker-u4Z3dw'
ERROR: Package 'pygments' requires a different Python: 2.7.17 not in '>=3.5'
```

**Comment by Anteru** [comment]
Thanks for raising this -- is using a local offline cache widespread? I can push out a 2.6.2 release without an universal wheel if that's causing problems, otherwise this will be fixed in a few months in the next bigger release.

**Comment by darkrain42** [comment]
Thanks for the quick follow-up!

It's probably not common..  I use it for a production app deploy, and am mixing both Python 2 and 3 packages while testing / cutting over to Python 3.

I've put this in my app's requirements.txt, which solves the issue, so I'm not in burning need of a fix:
```
# Workaround for pygments/pygments#1410
pygments < 2.6.0 ; python_version < '3.0'
```

**Comment by Anteru** [comment]
Ok, thanks :) I've fixed it for the next release. If enough issues crop up in 2.6 I'll do a point release with this fixed, if not it'll go into 2.7 which shouldn't be too far out.
